content/api/commands/selectfile.md

```diff
@@ -181,12 +186,13 @@ This will fail unless the file input has the `multiple` property.
 
 </Alert>
 
-### Selecting a file with custom filename and lastModified
+### Selecting a file with custom filename, mimeType and lastModified
 
 ```javascript
 cy.get('input[type=file]').selectFile({
   contents: 'path/to/file.json',
   fileName: 'custom-name.json',
+  mimeType: 'text/plain', // Overrides the 'application/json' mimeType detected from the .json extension
```

Comment:
I do like that this tells me which `mimeType` takes precedent FWIW. Might be good to have this comment live with the rest of the [described behavior](https://github.com/cypress-io/cypress-documentation/pull/4317/files#diff-e525be77ba3d25398f28f39f015e6719b03eb4b40aaa37404e480b5803abf6d5R73)?

Reply:
Doesn't the described behavior section explain that already with "If omitted, it will be inferred from the file extension"? Or do you mean to augment it with an example?

Reply:
Removing the comment, I agree that living with the rest of the described behavior is enough.